float64 axisX
float64 axisY
float64 axisZ

int32 hatX
int32 hatY

float64 throttle

bool[12] buttons
